airrohr-firmware/airrohr-firmware.ino

```diff
@@ -4674,9 +4680,11 @@ void setup(void) {
 	init_config();
 	init_display();
 	init_lcd();
-	setupNetworkTime();
-	connectWifi();
-	setup_webserver();
+	if(WIFI_enabled){
+		setupNetworkTime();
+		connectWifi();
+		setup_webserver();
```

Comment:
@MCLEANS we do still want to have the web server available especially to read values.